src/qtiCommonRenderer/renderers/interactions/InlineChoiceInteraction.js

```diff
@@ -63,11 +63,19 @@ var render = function(interaction, options) {
         $container.find('option[value=' + _emptyValue + ']').remove();
     }
 
+    const getItemDir = () => {
+        let item = $container.closest('.qti-item');
+        const itemBody = item.find('.qti-itemBody');
+        const itemDir = itemBody.attr('dir');
+        return itemDir;
+    }
+
     $container.select2({
         width: 'element',
         placeholder: opts.placeholderText,
         minimumResultsForSearch: -1,
-        dropdownCssClass: 'qti-inlineChoiceInteraction-dropdown'
+        containerCssClass: `${getItemDir()}`,
+        dropdownCssClass: `qti-inlineChoiceInteraction-dropdown ${getItemDir()}`
```

Comment:
thought: maybe it's an early optimization but you don't have to call `getItemDir` multiple times. DOM traversal like `closest` have a performance cost. I suggest you to 
```js
const dirClass = getItemDir();
$container.select2({
        width: 'element',
        placeholder: opts.placeholderText,
        minimumResultsForSearch: -1,
        containerCssClass: `${dirClass}`,
        dropdownCssClass: `qti-inlineChoiceInteraction-dropdown ${dirClass}`
 
```